lib/constants/http/http_constants.dart

```diff
@@ -0,0 +1,8 @@
+class HTTPConstants {
```

Comment:
We shouldn't commit tokens to the repo 😬

Reply:
I will create environment variables for tokens.